tinygrad/device.py

```diff
@@ -278,6 +279,9 @@ def get_linearizer(self, *ast:LazyOp) -> Linearizer:
         timed = sorted([(nm, tk, time_linearizer(tk, test_rawbuffers, allow_test_size=False, clear_l2=True)) for nm, tk in lins], key=lambda x: x[2])
         if DEBUG >= 1: print("  <  ".join(f"{nm:6s} : {lin.colored_shape(30, dense=True)} : {tm*1e6:8.2f} us" for nm, lin, tm in timed))
         k = timed[0][1]
+        if logkern and logkern_level > 1: logkern.write("\n".join(map(lambda x: str((x[1].ast, x[1].applied_opts,)), timed[1:]))+"\n")
```

Comment:
use `is not None` to compare with `None`. Also `writelines` to prevent manually add "\n"

Reply:
writelines doesn't append "\n", but this is still better than the having the previous join() call.